lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetColumnChunkIterator.java

```diff
@@ -13,112 +13,104 @@
  */
 package io.trino.parquet.reader;
 
-import io.airlift.slice.BasicSliceInput;
-import io.airlift.slice.Slice;
 import io.trino.parquet.DataPage;
 import io.trino.parquet.DataPageV1;
 import io.trino.parquet.DataPageV2;
 import io.trino.parquet.DictionaryPage;
 import io.trino.parquet.ParquetCorruptionException;
 import org.apache.parquet.column.Encoding;
-import org.apache.parquet.column.statistics.Statistics;
 import org.apache.parquet.format.DataPageHeader;
 import org.apache.parquet.format.DataPageHeaderV2;
 import org.apache.parquet.format.DictionaryPageHeader;
 import org.apache.parquet.format.PageHeader;
 import org.apache.parquet.format.Util;
 import org.apache.parquet.internal.column.columnindex.OffsetIndex;
 
+import javax.annotation.Nullable;
+
 import java.io.IOException;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.Iterator;
 import java.util.Optional;
 import java.util.OptionalLong;
 
-import static com.google.common.base.Verify.verify;
+import static com.google.common.base.Preconditions.checkArgument;
 import static io.trino.parquet.ParquetTypeUtils.getParquetEncoding;
 import static java.util.Objects.requireNonNull;
+import static org.apache.parquet.format.PageType.DICTIONARY_PAGE;
 
-public class ParquetColumnChunk
+public final class ParquetColumnChunkIterator
```

Comment:
There's a code comment in AbstractParquetDataSource that refers to `ParquetColumnChunk`, please update that to a @link on this class

Reply:
added just ParquetColumnChunkIterator. @link works only in Javadoc